temporal-opentracing/src/main/java/io/temporal/opentracing/internal/OpenTracingWorkflowOutboundCallsInterceptor.java

```diff
@@ -104,13 +237,16 @@ public <R> ChildWorkflowOutput<R> executeChildWorkflow(ChildWorkflowInput<R> inp
   public <R> ExecuteNexusOperationOutput<R> executeNexusOperation(
       ExecuteNexusOperationInput<R> input) {
     if (!WorkflowUnsafe.isReplaying()) {
+      Span capturedSpan = tracer.scopeManager().activeSpan();
       Span nexusOperationExecuteSpan =
           contextAccessor.writeSpanContextToHeader(
               () -> createStartNexusOperationSpanBuilder(input).start(),
               input.getHeaders(),
               tracer);
       try (Scope ignored = tracer.scopeManager().activate(nexusOperationExecuteSpan)) {
-        return super.executeNexusOperation(input);
+        ExecuteNexusOperationOutput<R> output = super.executeNexusOperation(input);
+        return new ExecuteNexusOperationOutput<>(
+            new PromiseWrapper<>(capturedSpan, output.getResult()), output.getOperationExecution());
```

Comment:
`output.getWorkflowExecution()` is a promise that needs to be wrapped

Reply:
Will fix

Reply:
fixed